### ISSUE TITLE
Finetune SVD with discrete time noise scheduler

### ISSUE THREAD

**Issue by HL-hanlin** [pixeli99/SVD_Xtend#35]
Hi, thanks for your great training code for SVD!

When I switch the default SVD sampler into discrete noise scheduler, the generated videos looks very bad. 
May I ask which layers/parameters of SVD should I finetune to make it compatible with discrete time noise scheduler? Or are there some ways that can transform the SVD noisy_latents (which is equal to latents + sigma * noise, with large sigma values) into corresponding noisy_latents (with smaller sigmas) used to train on discrete noise schedulers? 

Thanks!

**Comment by pixeli99** [comment]
You should need the FT timestep condition, you can refer to this part of the SVD paper for some help
![image](https://github.com/pixeli99/SVD_Xtend/assets/46072190/fcfe31b3-749f-4fcc-a314-3a3f7eb82fee)

**Comment by HL-hanlin** [comment]
Got it! Thanks for the pointer!
